src/components/ApplicationForm/Questionnaire.js

```diff
@@ -0,0 +1,152 @@
+            })
+          }
+          isValid
+        />
+        {formInputs.engagementSource === 'Other' && (
+          <TextInput
+            placeholder="Please Specify"
+            size="small"
+            noOutline
+            inline
+            onChange={e =>
+              onChange({
+                ...formInputs,
+                otherEngagementSource: e.target.value,
+              })
+            }
+          />
+        )}
+      </QuestionContainer>
+
+      <QuestionContainer>
+        <QuestionHeading>Question 15</QuestionHeading>
+        <H1 size="1.5em">Which nwPlus events have you been to? (Select all that apply)</H1>
+        <Select
+          type="checkbox"
+          label="Local Hack Day / Hack Camp"
+          checked={formInputs.eventsAttended.option1}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              multiselect: {
+                ...formInputs.eventsAttended,
+                option1: !formInputs.eventsAttended.option1,
+              },
+            })
+          }
+        />
+        <Select
+          type="checkbox"
+          label="nwHacks"
+          checked={formInputs.eventsAttended.option2}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              multiselect: {
+                ...formInputs.eventsAttended,
+                option2: !formInputs.eventsAttended.option2,
+              },
+            })
+          }
+        />
+        <Select
+          type="checkbox"
+          label="cmd-f"
+          checked={formInputs.eventsAttended.option3}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              multiselect: {
+                ...formInputs.eventsAttended,
+                option3: !formInputs.eventsAttended.option3,
+              },
+            })
+          }
+        />
+        <Select
+          type="checkbox"
+          label="cmd-f Phases"
+          checked={formInputs.eventsAttended.option4}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              multiselect: {
+                ...formInputs.eventsAttended,
+                option4: !formInputs.eventsAttended.option4,
+              },
+            })
+          }
+        />
+        <Select
+          type="checkbox"
+          label="nwPlus Workshop Series"
+          checked={formInputs.eventsAttended.option5}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              multiselect: {
+                ...formInputs.eventsAttended,
+                option5: !formInputs.eventsAttended.option5,
+              },
+            })
+          }
+        />
+        <Select
+          type="checkbox"
+          label="nwPlus Boothing"
+          checked={formInputs.eventsAttended.option6}
+          onChange={() =>
+            onChange({
+              ...formInputs,
+              multiselect: {
```

Comment:
multiselect doesn't seem to exist on the object you pass in, are you sure you named it right?

Reply:
it should look like this:
```javascript
          onChange={() =>
            onChange({
              ...formInputs,
              eventsAttended: {
  				...formInputs.eventsAttended,
                option6: !formInputs.eventsAttended.option6,
              },
            })

Reply:
Whoops you're totally right, thanks for the catch 🔥

Reply:
solved here a4316fc